internal/cmd/commands/ferry/addtoken.go

```diff
@@ -0,0 +1,187 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package ferry
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/boundary/api"
+	"github.com/hashicorp/boundary/internal/cmd/base"
+	"github.com/mitchellh/cli"
+	"github.com/posener/complete"
+)
+
+var (
+	_ cli.Command             = (*AddTokenCommand)(nil)
+	_ cli.CommandAutocomplete = (*AddTokenCommand)(nil)
+)
+
+type AddTokenCommand struct {
+	*base.Command
+}
+
+func (c *AddTokenCommand) Synopsis() string {
+	return "Add an auth token to a running Boundary ferry daemon"
+}
+
+func (c *AddTokenCommand) Help() string {
+	helpText := `
+Usage: boundary ferry add-token [options]
```

Comment:
Should we just make this `boundary ferry authenticate`?

Reply:
What is the use case for this command? As a user, sharing the token with Ferry should be automatic or triggered with `boundary authenticate` IMO. This feels like the wrong UX?

Reply:
I see. Do you think we need this for testing? I'd rather we implement multiplexing in Ferry before GA than need a command like this, so maybe we don't merge this new command?